lib/queueClient.ts

```diff
@@ -309,4 +331,44 @@ export class QueueClient extends Client {
   async cancelScheduledMessages(sequenceNumbers: Long[]): Promise<void> {
     return this._context.managementClient!.cancelScheduledMessages(sequenceNumbers);
   }
+
+  /**
+   * Lists the sessions on the ServiceBus Queue.
+   * @param skip The number of sessions to skip
+   * @param top Maximum numer of sessions.
+   * @param lastUpdateTime Filter to include only sessions updated after a given time. Default
+   * value: 3 days ago from the current time.
+   */
+  async listMessageSessions(skip: number, top: number, lastUpdatedTime?: Date): Promise<ListSessionsResponse> {
```

Comment:
What about subscription clients? Shouldnt they have session aware features like these? Or do we intend to cover that in a separate PR?